Source/Projections/Store/IProjectionStore.cs

```diff
@@ -14,6 +14,33 @@ namespace Dolittle.SDK.Projections.Store;
 /// </summary>
 public interface IProjectionStore
 {
+    /// <summary>
+    /// Gets the <see cref="IProjectionOf{TReadModel}"/> for the <typeparamref name="TProjection"/> projection read model class.
+    /// </summary>
+    /// <typeparam name="TProjection">The <see cref="Type" /> of the projection.</typeparam>
+    /// <returns>The <see cref="IProjectionOf{TReadModel}"/> for the <typeparamref name="TProjection"/> projection read model class.</returns>
+    IProjectionOf<TProjection> Of<TProjection>()
```

Comment:
Not sure if it even makes sense (cause you can't get the generic portion of the return type), but there could also be a `.Of(Type projection)` overload? Not sure if we have done this everywhere else...